src/js/util.js

```diff
@@ -928,7 +928,13 @@
             return element;
         },
 
+        // TODO: remove getFirstTextNode AND _getFirstTextNode when jumping in 6.0.0 (no code references)
         getFirstTextNode: function (element) {
+            Util.warn('getFirstTextNode is deprecated and will be removed in version 6.0.0');
+            return Util._getFirstTextNode(element);
+        },
+
+        _getFirstTextNode: function (element) {
```

Comment:
It's recursive, so each call to the method produces many many warning messages...which I found out when I ran the tests :smile: